examples/http_body/main.go

```diff
@@ -69,16 +69,14 @@ type setBodyContext struct {
 	// Embed the default root http context here,
 	// so that we don't need to reimplement all the methods.
 	types.DefaultHttpContext
-	modifyResponse        bool
-	totalRequestBodySize  int
-	totalResponseBodySize int
-	bufferOperation       string
+	modifyResponse  bool
+	bufferOperation string
 }
 
 // Override types.DefaultHttpContext.
 func (ctx *setBodyContext) OnHttpRequestHeaders(numHeaders int, endOfStream bool) types.Action {
 	mode, err := proxywasm.GetHttpRequestHeader("buffer-replace-at")
-	if mode == "response" {
+	if err == nil && mode == "response" {
```

Comment:
curious what's the reason for this check?

Reply:
`err` is retrieved but not used. I was in doubt about whether not even retrieve it, or to ensure it was nil before relying on `mode`